bedrock/externalpages/templates/externalpages/pocket/includes/nav.html

```diff
@@ -0,0 +1,50 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+<header class="pocket-header">
+    <nav class="global-nav-container">
+        <button class="global-nav-mobile-menu-btn inline">
```

Comment:
This button is visible on smaller viewports, but clicking it doesn't do anything. Are we missing some JS from these pages?

Reply:
Not sure about this, really. @nathan-barrett may have a better answer!